scenes/game/game.gd

```diff
@@ -2868,6 +2879,7 @@ func _handle_events(events):
 			Enums.EventType.EventType_Strike_Response:
 				_on_strike_started(event, false)
 			Enums.EventType.EventType_Strike_Response_Ex:
+				_add_bonus_label_text(event['event_player'], "EX Strike")
```

Comment:
Rather than doing this, I think I'd prefer adding like some wordart "EX" on the top right corner of the card that pops up.
Though to do that we should also probably have that show up for boosts that make it EX and then it also needs to go away if it becomes not EX.
This is probably fine for now? But if you want to make it look really good, probably do this?